src/data/astarte_message_hub_node.rs

```diff
@@ -236,10 +242,17 @@ mod tests {
 
         let (server_handle, drop_sender, port) = run_local_server(msg_hub).await;
 
+        let (store, tmp_store_path) = create_tmp_store().await;
+        let tmp_inrtefaces_path =
+            TempDir::new("edgehog-tmp-interfaces").expect("failed to create interface tmp dir");
+
```

Comment:
We can use the store directory here

Reply:
Since we know it exists and is empty (without JSON files)

Reply:
Sure it make sense. Since it is a test, we can do this way